app/Entirety/devices/templates/devices/list.html

```diff
@@ -6,109 +6,205 @@
 
 {% block content %}
 
-<div class="d-flex">
-    <span class="h3">Devices </span>
-</div>
+{#<div class="d-flex">#}
+{#    <span class="h3">Devices </span>#}
+{#</div>#}
 
-<form action="" method="GET" >
-        {% csrf_token %}
-     <div class="row justify-content-center">
-         <div class="col-8">
-            <div class="p-3 border border-light rounded-pill bg-light">
-                    <div class="input-group border-0 shadow-none">
-                        <input name="search-patern" type="text" class="form-control border-light rounded-pill"
-                               placeholder="Filter devices ..."/>
-                        <span class="input-group-btn">
-                            <button class="btn btn-info btn-lg bg-light border-light" type="submit"
-                                    value="search">
-                                <i class="bi-search"></i>
-                            </button>
-                        </span>
-                    </div>
-            </div>
+<ul class="nav nav-tabs">
+    {% if to_servicegroup %}
+        <li class="nav-item">
+            <a class="nav-link" data-bs-toggle="tab" href="#devices">Devices</a>
+        </li>
+        <li class="nav-item">
+            <a class="nav-link active" data-bs-toggle="tab" href="#groups">Services Group</a>
+        </li>
+    {% else %}
+        <li class="nav-item">
+            <a class="nav-link active" data-bs-toggle="tab" href="#devices">Devices</a>
+        </li>
+        <li class="nav-item">
+            <a class="nav-link" data-bs-toggle="tab" href="#groups">Services Group</a>
+        </li>
+    {% endif %}
+</ul>
+
+<div class="tab-content">
+  <br>
+  {% if to_servicegroup %}
```

Comment:
This block can be replaced with a single line like: `<div {% if to_servicegroup %} class="tab-pane container fade" {% else %} class="tab-pane container active" {% endif %} id="devices">`